README.md

```diff
@@ -242,6 +242,7 @@ Known packages needed on Fedora-based systems:
  * `libpq-devel`
  * `openssl-devel`
  * `systemd-devel`
+ * `libsodium-dev`
```

Comment:
Hmm, isn't also the `libsodium` required ? (the one without `dev` suffix)

Reply:
libsodium will be a dependency of libsodium-dev